docs/setup/settings.asciidoc

```diff
@@ -450,7 +450,7 @@ responses to the client from the {kib} server, and specifies what value is used.
 Header names and values to send on all responses to the client from the {kib} server. *Default: `{}`*
 
 [[server-shutdownTimeout]] `server.shutdownTimeout`::
-Sets the grace period for {kib} to attempt to resolve any ongoing HTTP requests after receiving a `SIGTERM`/`SIGINT` signal, and before shutting down. Any new HTTP requests received during this period are rejected with a `503` response. *Default: `30s`*
+Sets the grace period for {kib} to attempt to resolve any ongoing HTTP requests after receiving a `SIGTERM`/`SIGINT` signal, and before shutting down. Any new HTTP requests received during this period are rejected (the incoming socket is closed without further processing). *Default: `30s`*
```

Comment:
```suggestion
Sets the grace period for {kib} to attempt to resolve any ongoing HTTP requests after receiving a `SIGTERM`/`SIGINT` signal, and before shutting down. Any new HTTP requests received during this period are rejected, because the incoming socket is closed without further processing. *Default: `30s`*
```
Thank you for updating the docs as well 🙏, very very minor styling edit to avoid using parenthesis.